content/models/social_force_model/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Social Force Model
+math: true
+---
+
+## Introduction to Social Force Model
+
+The Social Force Model [[1]](#Helbing2000) is a force-based model that defines the movement of pedestrians by the combination of different social forces affecting an individual. 
+
+There are three basic forces that affect an individual. The driving force represents a person's desire to move in a certain direction, independent of other people and obstacles. The agent force is caused by the interaction between the individuals and causes them to avoid each other in order to avoid collisions. The obstacle force acts in a similar way to the person force to avoid collisions with obstacles in the environment.
+
+## Driving force
+The driving force defines the force that propels pedestrians towards their intended destinations:
+
+$$ \overrightarrow{F_i^{\mathrm{drv}}}=\frac{v_i^0\overrightarrow{e_i^0} - \overrightarrow{v_i}}{\tau}$$
+* $v_i^0$ is the desired speed
+* $\overrightarrow{e_i^0}$ is the desired direction
+* $v_i$ is the currect velocity
+* $\tau$ is the reaction time
+
+{{< figure src="Driving_Force.png" width="500">}}
+
+In the figure above, the black dot represents an agent. Its radius is represented by the gray circle around it. The desired direction is towards the target, which is indicated by an X.
+
+## Agent force
+The agents exert a repulsive force on each other, which increases exponentially with their proximity. When they collide the pushing forces are increased additionally. When colliding, a frictional force also occurs, which acts orthogonally to the repulsive force in the direction of the speed difference. 
+
+$$ \overrightarrow{F\_i^{\mathrm{ped}}} = \sum_{j} \overrightarrow{f_{ij}}$$
+
+$$\overrightarrow{f_{ij}} = [A_i exp[(r_{ij} - d_{ij})/B_i] + kg(r_{ij} - d_{ij})] \overrightarrow{n_{ij}} + \kappa g(r_{ij} - d_{ij}) \Delta v\_{ji}^t \overrightarrow{t_{ij}}$$
+
+$$\phantom{......}\fbox{\phantom{...................} pushing \phantom{...................}} \fbox{\phantom{......} sliding \phantom{......}}$$
+
+* $j$ is another agent
+* $r_{ij} = r_i + r_j$ is the combined radius of $i$ and $j$
+* $d_{ij}$ is the distance between $i$ an $j$ them. 
+* $A_i$, $B_i$, $k$ and $\kappa$ are constants 
+  -  $A_i$ agent scale
+  -  $B_i$ force distance
+  -  $k$ body force
+  -  $\kappa$ friction
+* $g$ is a function with $g(x) =  \begin{cases}
```

Comment:
$g$ represents the distance between pedestrians when they are in contact, and is zero when there is no contact.